src/Module/Admin/Storage.php

```diff
@@ -78,7 +78,7 @@ public static function post(array $parameters = [])
 
 		if (!empty($_POST['submit_save_set'])) {
 			try {
-				/** @var IWritableStorage $newstorage */
+				/** @var \Friendica\Core\Storage\Capability\ICanWriteToStorage $newstorage */
```

Comment:
Extraneous absolute name:
```suggestion
				/** @var ICanWriteToStorage $newstorage */
```